src/components/PhoneNumberInput/index.js

```diff
@@ -0,0 +1,45 @@
+import { h, Component } from 'preact'
+import PhoneNumber, {isValidPhoneNumber} from 'react-phone-number-input'
+import 'react-phone-number-input/rrui.css'
+import 'react-phone-number-input/style.css'
+
+import style from './style.css'
+
+class PhoneNumberInput extends Component {
+  constructor(props) {
+    super(props)
+    this.state = { value: null}
+  }
+
+  onChange = (value) => {
+    this.setState({value})
+    this.props.clearPreviousAttempts()
+    this.updateValidNumber()
+  }
+
+  updateValidNumber = () => {
+    const number = this.state.value
+    const validNumber = isValidPhoneNumber(number)
+    if (number && validNumber) {
+      this.props.updateNumber(this.state.value)
+    }
+  }
+
+  customStyle = {
```

Comment:
It looks like `react-phone-number-input` produces BEM-style global class names.

It might be better to style those through your `style.css` rather than passing in a style object

Mixing CSS-in-JS and CSS modules gets a bit messy after a while